components/dashboard/src/settings/Plans.tsx

```diff
@@ -629,101 +631,99 @@ export default function () {
         );
     }
 
-    const showPlans = userBillingMode && userBillingMode.mode === "chargebee";
     return (
         <div>
             <PageWithSettingsSubMenu title="Plans" subtitle="Manage account usage and billing.">
-                {showPlans && (
-                    <div className="w-full text-center">
-                        <p className="text-xl text-gray-500">
-                            You are currently using the{" "}
-                            <span className="font-bold">
-                                {Plans.getById(assignedTs?.planId)?.name || currentPlan.name}
-                            </span>{" "}
-                            plan.
+                {isUsageBasedBillingEnabled && (
+                    <Alert type="message" className="mb-4">
+                        Your account has been enabled for usage-based billing. Discover faster workspace classes and
+                        only pay for what you actually use.{" "}
+                        <a className="gp-link" href="https://www.gitpod.io/docs/configure/billing/usage-based-billing">
+                            Learn more
+                        </a>
+                        <br />
+                        <br />
+                        The old monthly plans are deprecated and will be cancelled by End of March 2023.
+                    </Alert>
```

Comment:
Hey Jan! Some issues on the copy used here:
1. Too verbose, or uses unnecessary empty space.
2. Focus on the personal account, not [team plans](https://www.gitpod.io/docs/configure/billing/team-plans) or [team billing](https://www.gitpod.io/docs/configure/billing/team-billing).
3. Sounds like the user has been part of a closed beta list and not accessing a GA feature on Gitpod.io.
5. Voice and tone is not direct, using terms like _discover_. 

What do you think of the following copy?

| BEFORE | AFTER |
|-|-|
| <img width="756" alt="alert-before" src="https://user-images.githubusercontent.com/120486/205722636-fd748bc8-97b7-4abf-bd55-a50a7eac7628.png"> | <img width="756" alt="alert-after" src="https://user-images.githubusercontent.com/120486/205722653-7fcd2c15-1261-49ae-af6b-7de553a19601.png"> |

Reply:
Let's not call our pricing as a whole "pay-as-you-go". the pay as you go bit is only the mode where you pay later. We have a free plan and a personal plan with a certain amount of credits that is not pay-as-you-go. We'll likely have other ways to prepurchase credits in the future. 
"Usage-based" is ok but I'd prefer to generally just talk about (new) pricing.

Reply:
DEAL 🤝